app/views/admin/listing_shapes/index.haml

```diff
@@ -38,7 +38,7 @@
         - elsif shape[:category_ids].size == category_count
           = t("admin.listing_shapes.index.all_categories")
         - else
-          = "#{shape[:category_ids].size}/#{category_count}"
+          = t("admin.listing_shapes.index.category_count", :count => "#{shape[:category_ids].size}/#{category_count}")
```

Comment:
I guess this works ok, but I'd be a careful using the variable name `count` here. `count` has a special meaning for pluralizations:

```yml
apples:
  one: %{count} apple
  other: %{count} apples
```

```ruby
t('apples', count: 5) #=> 5 apples
```

http://guides.rubyonrails.org/i18n.html#pluralization